src/app/components/google-map-view/map.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from "react";
+import { withScriptjs, withGoogleMap, GoogleMap } from "react-google-maps/types";
+import { withProps, compose } from "recompose";
+
+import MarkerClusterer from "react-google-maps/lib/components/addons/MarkerClusterer";
+import Marker from "react-google-maps/lib/components/Marker";
+
+import { demoMapStyle } from "./map-style";
+import { Markers } from "./markerList";
+
+const GOOGLE_MAPS_API_KEY = "AIzaSyCUfG5xqkrkJp4s_2nPnPNPPd3FfASE-sI";
```

Comment:
Move `const GOOGLE_MAPS_API_KEY` to another location as a separate file and add to `.gitignore` to not push into Github.